app/code/community/Bolt/Boltpay/etc/system.xml

```diff
@@ -480,10 +480,28 @@ This function is called when the Bolt checkout modal is closed.
               <show_in_store>1</show_in_store>
               <frontend_class>validate-number</frontend_class>
             </sort_order>
+            <datadog_key translate="label">
+              <label>DataDog API Key</label>
+              <frontend_type>text</frontend_type>
+              <sort_order>150</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>1</show_in_store>
+            </datadog_key>
+            <datadog_key_severity translate="label">
```

Comment:
This requirement also came from Kazuki. See here https://prnt.sc/mihnl2. I think the purpose was that we would set it to only return errors unless we start seeing problems, and then we would turn on more thorough debugging as this could likely reduce server performance if it's getting hit 9-10 times during a request. I could also change this to be a hidden value, but I do think it would be good to make it configurable from the admin.

Reply:
@johnmccombs1 Yes, the purpose of the options and configuration and the reasoning behind it is self-evident for me.  My thoughts and concern is making this such a forward facing config for a merchant who will then look and say "What the hell is Data Dog?"  It reminds me of some long conversations that I had to explain about `track.js`.

Moreover, we don't even want merchants accidentally change our default settings or even think about doing it.  It seems like changes to this, especially changing the key, should have a very clear cut reason. I think making these as hidden options fulfill the requirement for which Kazuki is asking.

What do you think?

By the way, perhaps I'm reading the code wrong, but it seems to me that by default you are turning on full logging.

https://github.com/BoltApp/bolt-magento1/blob/f32700066251854f7dac0d683135bfcfe7b24ce7/app/code/community/Bolt/Boltpay/etc/config.xml#L276

If I have read the code correctly, would this be a better default? 
`<datadog_key_severity>error</datadog_key_severity>`

Reply:
I'm in complete agreement. I'll update it.